### PR TITLE
[CMD] Improve Indonesian (ID-id) translation

### PR DESCRIPTION
**Pull request by reinhart1010**
Improve Indonesian (ID-id) translation for several CMD links:

+ Change "version" to "versi"
+ Fully translate the "color" command help text (the former translation version still have a bunch of English words)
+ Consider using more formal words such as "dapat" instead of "bisa", and "...di bawah ketentuan lisensi GNU General Public License sebagaimana..." instead of "...di bawah term lisensi GNU General Public License seperti..." (**Original text:** ...under the terms of GNU General Public License as...")

### CODE REVIEW

File: base/shell/cmd/lang/id-ID.rc [hunk 1]
```diff
@@ -100,9 +100,9 @@ CTTY device\n\n\
             CON is usually the default standard I/O device.\n\n\
 To return control to the standard console, type: CTTY CON on the auxiliary\n\
 device."
-    STRING_DATE_HELP1 "\nMasukkan tanggal baru (mm%cdd%cyyyy): "
-    STRING_DATE_HELP2 "\nMasukkan tanggal baru (dd%cmm%cyyyy): "
-    STRING_DATE_HELP3 "\nMasukkan tanggal baru (yyyy%cmm%cdd): "
+    STRING_DATE_HELP1 "\nMasukkan tanggal baru (format: mm%cdd%cyyyy): "
```
Comment (Mas4hmad) [inline base/shell/cmd/lang/id-ID.rc:103]: preferred to keep this. (regarding to other language resource format)
```suggestion
    STRING_DATE_HELP1 "\nMasukkan tanggal baru (mm%cdd%cyyyy): "
```
including two following line below

Comment (reinhart1010) [reply base/shell/cmd/lang/id-ID.rc]: Reverted in https://github.com/reactos/reactos/pull/3417/commits/60f67146b89693a5d391a9481a30f7965dae4df5

**Review by Mas4hmad — APPROVED** [review]

**Review by Fraizeraust — CHANGES_REQUESTED** [review]
The commit uses your nickname instead of the real name. Please change it and amend the commit.
Capture

**Comment by Mas4hmad** [comment]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit/3042512#3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):

```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:

```
git config user.name "Your Name"
git config user.email you@example.com
```

Also please update your name in [GitHub profile settings](https://github.com/settings/profile).

Regards,

**Comment by reinhart1010** [comment]
I have attempted to amend the original commits to fix the author name.

However, it seems that the culprit comes from the GitHub website (e.g. when accepting PRs and committing the file directly on the website), for example reinhart1010/reactos#1 to merge newer ReactOS commits from `master`.

image

Here's the current profile settings (to be changed soon to remove initials)

image

**Comment by binarymaster** [comment]
Your branch was messed up a bit with merge commits, I fixed it to preserve linear history.

@Mas4hmad the new changes have been added since your last approval, please review again.

**Review by Mas4hmad — APPROVED** [review]
no build problem and change requested for the translation
it's all okay for me to this PR to be merged.

**Comment by binarymaster** [comment]
Thanks!

**Pull request merged**